storm-core/src/jvm/org/apache/storm/trident/windowing/config/WindowConfig.java

```diff
@@ -31,13 +31,13 @@
      * Returns the length of the window.
      * @return
      */
-    public int getWindowLength();
+    public long getWindowLength();
```

Comment:
If we want to change counts also to `Long` then `SlidingCountWindow` and `TumblingCountWindow` should be changed respectively. But there is no real need to have `Long` for count based windowing.